applications/crossbar/src/modules/cb_recordings.erl

```diff
@@ -19,7 +14,9 @@
         ,resource_exists/0, resource_exists/1
         ,content_types_provided/2
         ,validate/1, validate/2
-        ,delete/2
+        ,delete/2, patch/2
+        ,to_json/1
+        ,to_csv/1
```

Comment:
Does the API not respond with CSV when you use `Accept: text/csv`? I don't think this is needed unless there's something off with the generic CSV generator.